modules/gltf/gltf_document.cpp

```diff
@@ -613,12 +615,10 @@ Error GLTFDocument::_parse_nodes(Ref<GLTFState> p_state) {
 			if (n.has("scale")) {
 				node->set_scale(_arr_to_vec3(n["scale"]));
 			}
-
-			Transform3D godot_rest_transform;
 			godot_rest_transform.basis.set_quaternion_scale(node->transform.basis.get_rotation_quaternion(), node->transform.basis.get_scale());
 			godot_rest_transform.origin = node->transform.origin;
-			node->set_additional_data("GODOT_rest_transform", godot_rest_transform);
 		}
+		node->set_additional_data("GODOT_rest_transform", godot_rest_transform);
```

Comment:
I would delete `godot_rest_transform` and then do:
```suggestion
		node->set_additional_data("GODOT_rest_transform", node->transform);
```

Reply:
Weird TRS is desynced from matrix.

Reply:
Or at least asymmetrical.

Reply:
```
Vector3 GLTFNode::get_position() {
	return transform.origin;
}

void GLTFNode::set_position(Vector3 p_position) {
	transform.origin = p_position;
}

Quaternion GLTFNode::get_rotation() {
	return transform.basis.get_rotation_quaternion();
}

void GLTFNode::set_rotation(Quaternion p_rotation) {
	transform.basis.set_quaternion_scale(p_rotation, transform.basis.get_scale());
}

Vector3 GLTFNode::get_scale() {
	return transform.basis.get_scale();
}

void GLTFNode::set_scale(Vector3 p_scale) {
	transform.basis = transform.basis.orthonormalized() * Basis::from_scale(p_scale);
}
```

@aaronfranke modified the code a few years to avoid two ways of specifying the same transform. Looks good.